services/venice-server/src/main/java/com/linkedin/venice/stats/ServerConnectionStats.java

```diff
@@ -0,0 +1,43 @@
+package com.linkedin.venice.stats;
+
+import io.tehuti.metrics.MetricsRepository;
+import io.tehuti.metrics.Sensor;
+import io.tehuti.metrics.stats.Max;
+
+
+public class ServerConnectionStats extends AbstractVeniceStats {
+  public static final String ROUTER_CONNECTION_COUNT_GAUGE = "router_connection_count_gauge";
```

Comment:
We don't need to put `gauge` into the metric name since `Gauge` type will be added to the metric name automatically.

Reply:
Right, let me fix that.